start.js

```diff
@@ -228,6 +228,10 @@ env.addFilter('formatdate', function(str) {
   return `${date.getDate()} ${monthNames[date.getMonth()]} ${date.getFullYear()}`
 });
 
+env.addFilter('where', function (array, key, value) {
```

Comment:
Could call this filter “find” to match the JavaScript function? “Where” to me implies “get all matching” but the find function only returns the first one.